docs/azure_arc_jumpstart/azure_arc_data/kubeadm/kubeadm_azure_dc_vanilla_arm_template/_index.md

```diff
@@ -112,6 +112,21 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
   - _`deployPostgreSQL`_ - Boolean that sets whether or not to deploy PostgreSQL, for this data controller vanilla scenario we leave it set to _**false**_.
   - _`deployBastion`_ - Choice (true | false) to deploy Azure Bastion or not to connect to the client VM.
 
+- You will also need to get the Azure Custom Location Resource Provider ([RP](https://learn.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types)) Object ID (OID) and export it as an environment variable:
```

Comment:
- You already had a Resource Provider hyperlink in the first appearance, no need to dup. 
- Would be good to have one line on why the OID is needed and what it is being used for.